_sass/syntax-highlighting/github.scss

```diff
@@ -0,0 +1,301 @@
+/**
+ * Syntax highlighting styles
+ */
+
+.highlight,
+.highlighter-rouge {
+    color: #4271ae;
+    margin-bottom: 2rem;
+}
+
+.highlight > pre,
+.highlighter-rouge > pre {
+    font-size: 13px;
+    line-height: 1.5;
+    color: #444;
+    -moz-tab-size: 4;
+    tab-size: 4;
+    overflow: auto;
+    padding: 25px;
+    border: none;
+    border-radius: 0;
+    background: #f6f8fa;
+}
+
+.highlight {
+    /*
+      * GitHub style for Pygments
+      * Courtesy of GitHub.com
+      */
+    .hll {
+        background-color: #f8f8f8;
+        border: 1px solid #ccc;
+        padding: 6px 10px;
+        border-radius: 3px;
+    }
+
+    .c {
+        color: #999988;
+        font-style: italic;
+    }
+
+    .err {
+        color: #a61717;
+        background-color: #e3d2d2;
+    }
+
+    .k {
+        font-weight: bold;
+    }
+
+    .o {
+        font-weight: bold;
+    }
+
+    .cm {
+        color: #999988;
+        font-style: italic;
+    }
+
+    .cp {
+        color: #999999;
+        font-weight: bold;
+    }
+
+    .c1 {
+        color: #999988;
+        font-style: italic;
+    }
+
+    .cs {
+        color: #999999;
+        font-weight: bold;
+        font-style: italic;
+    }
+
+    .gd {
+        color: #000000;
+        background-color: #ffdddd;
+    }
+
+    .gd .x {
+        color: #000000;
+        background-color: #ffaaaa;
+    }
+
+    .ge {
+        font-style: italic;
+    }
+
+    .gr {
+        color: #aa0000;
+    }
+
+    .gh {
+        color: #999999;
+    }
+
+    .gi {
+        color: #000000;
+        background-color: #ddffdd;
+    }
+
+    .gi .x {
+        color: #000000;
+        background-color: #aaffaa;
+    }
+
+    .go {
+        color: #888888;
+    }
+
+    .gp {
+        color: #555555;
+    }
+
+    .gs {
+        font-weight: bold;
+    }
+
+    .gu {
+        color: #800080;
+        font-weight: bold;
+    }
+
+    .gt {
+        color: #aa0000;
+    }
+
+    .kc {
+        font-weight: bold;
+    }
+
+    .kd {
+        font-weight: bold;
+    }
+
+    .kn {
+        font-weight: bold;
+    }
+
+    .kp {
+        font-weight: bold;
+    }
+
+    .kr {
+        font-weight: bold;
+    }
+
+    .kt {
+        color: #445588;
+        font-weight: bold;
+    }
+
+    .m {
+        color: #009999;
+    }
+
+    .s {
+        color: #dd1144;
+    }
+
+    .n {
+        color: #333333;
+    }
+
+    .na {
+        color: teal;
+    }
+
+    .nb {
+        color: #0086b3;
+    }
+
+    .nc {
+        color: #445588;
+        font-weight: bold;
+    }
+
```

Comment:
I've not used sass before but this does seem like a lot of styles to commit! Are these things that can be pulled in on site build instead of being committed? Or can they be reduced/minified somehow? This is my ignorance though, so I'm happy to keep them if this is standard.

Reply:
You are right! I
ll look into this further tomorrow. Also great news for the form, I found a plug in that is able to send form submissions to a slack channel!  https://help.formspree.io/hc/en-us/articles/360045648933-Send-Slack-notifications

I got the form working too.